test/Mono.Linker.Tests.Cases/UnreachableBlock/BodiesWithSubstitutions.cs

```diff
@@ -1,13 +1,21 @@
 using System.Runtime.CompilerServices;
 using Mono.Linker.Tests.Cases.Expectations.Assertions;
 using Mono.Linker.Tests.Cases.Expectations.Metadata;
+using Mono.Linker.Tests.Cases.UnreachableBlock.Dependencies;
 
 namespace Mono.Linker.Tests.Cases.UnreachableBlock
 {
 	[SetupLinkerSubstitutionFile ("BodiesWithSubstitutions.xml")]
 	[SetupCSharpCompilerToUse ("csc")]
 	[SetupCompileArgument ("/optimize+")]
 	[SetupLinkerArgument ("--enable-opt", "ipconstprop")]
+	[SetupCompileBefore (
+		"LibReturningConstant.dll", 
+		new string[] { "Dependencies/LibReturningConstant.cs" })]
+	[SetupCompileBefore (
+		"LibWithConstantSubstitution.dll", 
+		new string[] { "Dependencies/LibWithConstantSubstitution.cs" }, 
+		resources: new string[] { "Dependencies/LibWithConstantSubstitution.xml" })]
```

Comment:
I'm not sure why it doesn't want the string [] in the resources, changing it to object[] like is being used in other parts of the code solve it.

Reply:
Thanks for the suggestion - unfortunately it didn't help :-(.